index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
@joshuaogle I've provided two references that say pretty clearly that "alright" is incorrect (that is, it's not an "alternative", merely an error). If you're going to claim otherwise, the least you could do is provide a reference supporting your point of view.